src/metadataGeneration/tsoa.ts

```diff
@@ -14,7 +14,7 @@ export namespace Tsoa {
   export interface Method {
     deprecated?: boolean;
     description?: string;
-    method: 'get' | 'post' | 'put' | 'delete' | 'options' | 'head' | 'patch' | 'head';
+    method: 'get' | 'post' | 'put' | 'delete' | 'options' | 'head' | 'patch';
```

Comment:
Was this removed intentionally?

Reply:
Yes, there was duplicate in enumeration.

Reply:
Good point :laughing: